artifacts/openapi.yaml

```diff
@@ -5381,6 +5386,31 @@ components:
           maximum: 16777215
           default: 10
           x-field-uid: 4
+    Bgp.UpdateReplay:
+      description: |-
+        Ordered BGP Updates ( including both Advertise and Withdraws ) to be sent to the peer after the BGP session is established.
+      type: object
+      properties:
+        updates:
+          description: "Array of ordered BGP Updates ( including both Advertise and\
+            \ Withdraws ) to be sent to the peer after the BGP session is established. "
+          type: array
+          items:
+            $ref: '#/components/schemas/Bgp.OneUpdateReplay'
+          x-field-uid: 1
+    Bgp.OneUpdateReplay:
+      description: "Specification of one BGP Update to be sent to the BGP peer.  \
+        \      "
+      type: object
+      properties:
+        update_bytes:
+          description: |-
+            Bytes to be sent to peer after the BGP Update Header. The Update Header will always have the initial 16 bytes containing  Marker bytes, 2 bytes containing the Length and 1 byte containing the Type.The string MUST contain sequence of valid hex bytes. The bytes specified should be appended to  the Update message to be sent to the peer after the fixed 19 bytes above.  The maximum length of 4077 is obtained from the maximum length allowed for a BGP message in RFC4271 which is 4096 minus mandatory  19 bytes described above.
```

Comment:
While talking about the maximum, talk about the minimum also.

Reply:
I specifically mentioned the logic for maximum since without this 4077 might appear to be an arbitrary max. Either way, I have added that the minimum length of the input byte stream should be 1 as per previous comment since we are allowing the user to send whatever he/she wants upto max length.